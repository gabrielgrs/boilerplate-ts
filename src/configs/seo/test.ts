import seoConfigs from './index'

describe('SEO configs', () => {
  it('Seo configs should match snapshot', () => {
    expect(seoConfigs).toMatchSnapshot()
  })
})
