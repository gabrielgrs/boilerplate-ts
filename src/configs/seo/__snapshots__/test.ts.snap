// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SEO configs Seo configs should match snapshot 1`] = `
Object {
  "openGraph": Object {
    "locale": "pt_BR",
    "site_name": "BoilerplateTS",
    "title": "BoilerplateTS",
    "type": "website",
    "url": "",
  },
}
`;
